app/app.go

```diff
@@ -453,3 +459,20 @@ func GetMaccPerms() map[string][]string {
 
 	return dupMaccPerms
 }
+
+// Required for ibctesting
+func (app *OsmosisApp) GetStakingKeeper() stakingkeeper.Keeper {
+	return *app.AppKeepers.StakingKeeper
+}
```

Comment:
Yikes, this actually looks to me like a bug in ibctesting, that it depends on a keeper struct return arg, instead of an interface

Reply:
issue made https://github.com/cosmos/ibc-go/issues/1971

Reply:
yeah, it doesn't seem like reuse was the main priority in ibctesting, but it's easy enough to override stuff once you know where to look :)